dt-core/admin/menu/tabs/tab-custom-lists.php

```diff
@@ -642,13 +639,13 @@ private function languages_box(){
                 <tbody>
                 <?php foreach ( $languages as $language_key => $language_option ) :
 
-                    $enabled = !isset( $language_option['deleted'] ) || $language_option['deleted'] == false; ?>
+                    $enabled = !isset( $language_option['enabled'] ) || $language_option['enabled'] == true; ?>
                     
                     <tr class="language-row" data-lang="<?php echo esc_html( $language_key ) ?>">
                         <td class='lang_key'><?php echo esc_html( $language_key ) ?></td>
                         <td class='default_label'><?php echo esc_html( isset( $dt_global_languages_list[$language_key] ) ? $dt_global_languages_list[$language_key]['label'] : '' ) ?></td>
-                        <td class='custom_label'><input type="text" placeholder="Custom Label" name="language_label[<?php echo esc_html( $language_key ) ?>][default]" value="<?php echo esc_html( ( !isset( $dt_global_languages_list[$language_key] ) || ( isset( $dt_global_languages_list[$language_key] ) && $dt_global_languages_list[$language_key]['label'] != $language_option['label'] ) ) ? $language_option['label'] : '' ) ?>"></td>
-                        <td class='iso_code'><input type="text" placeholder="ISO 639-3 code" maxlength="3" name="language_code[<?php echo esc_html( $language_key ) ?>]" value="<?php echo esc_html( $language_option['iso_639-3'] ?? '' ) ?>"></td>
+                        <td class='custom_label'><input type="text" placeholder="Custom Label" name="language_label[<?php echo esc_html( $language_key ) ?>][default]" value="<?php echo esc_html( $language_option['label'] ?? '' )?>"></td>
```

Comment:
@CptHappyHands keep the "custom" label blank if the admin has not saved a custom label yet. This should not be filled with the default label.